components/automate-ui/src/app/pages/+compliance/+reporting/reporting.component.ts

```diff
@@ -460,7 +463,7 @@ export class ReportingComponent implements OnInit, OnDestroy {
 
     reportQuery.filters = urlFilters.filter(
       (urlParm: Chicklet) => this.isTypeControlTag(urlParm.type)
-        || this.allowedURLFilterTypes.indexOf(urlParm.type) >= 0)
+        || this.allowedURLFilterTypes.indexOf(urlParm.type as ReportingFilterTypes) >= 0)
```

Comment:
Same here with urlParm.type actually being part of the Enum `ReportingFilterTypes`